quiz.css

```diff
@@ -0,0 +1,78 @@
+article {
+  width: 30%;
+  border: 1px solid lightblue;
+  font-family: sans-serif;
+  float: left;
+  margin: 1%;
+}
+
+article h1 {
+  border: 1px dotted lightgray;
+  text-align: center;
+}
```

Comment:
You are over specific. Unless you have different styles for different h1s, simply use h1